policy-test/tests/e2e_egress_network.rs

```diff
@@ -0,0 +1,692 @@
+use linkerd_policy_controller_k8s_api as k8s;
+use linkerd_policy_test::{
+    await_condition, await_gateway_route_status, await_tcp_route_status, await_tls_route_status,
+    create, create_ready_pod, curl, endpoints_ready, update, web, with_temp_ns, LinkerdInject,
+};
+
+#[tokio::test(flavor = "current_thread")]
+async fn default_traffic_policy_http() {
+    with_temp_ns(|client, ns| async move {
+        let mut egress = create(
+            &client,
+            k8s::policy::EgressNetwork {
+                metadata: k8s::ObjectMeta {
+                    namespace: Some(ns.clone()),
+                    name: Some("egress".to_string()),
+                    ..Default::default()
+                },
+                spec: k8s::policy::EgressNetworkSpec {
+                    traffic_policy: k8s::policy::TrafficPolicy::Allow,
+                    networks: None,
+                },
+                status: None,
+            },
+        )
+        .await;
```

Comment:
Would it make sense to await for the egressnetwork to become accepted before attempting to send traffic?

Reply:
Yes this makes sense. We should do that.